### PR TITLE
chore(css): Move CSS examples - Flexbox, Flow

### PR DESCRIPTION
**Pull request by bsmth**
### Description

As part of an initiative to reduce repo maintenance & code duplication, this PR moves some external examples into content. This PR converts the following macros to live samples:

```
{{EmbedGHLiveSample("css-examples/flexbox/basics/simple-example.html", '100%', 600)}}
{{EmbedGHLiveSample("css-examples/flexbox/wrapping/row-wrap.html", '100%', 650)}}
{{EmbedGHLiveSample("css-examples/flexbox/wrapping/column-wrap.html", '100%', 810)}}
{{EmbedGHLiveSample("css-examples/flexbox/wrapping/row-reverse-wrap.html", '100%', 750)}}
{{EmbedGHLiveSample("css-examples/flexbox/wrapping/grid-example.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/flexbox/wrapping/flex-grid.html", '100%', 650)}}
{{EmbedGHLiveSample("css-examples/flexbox/wrapping/gaps.html", '100%', 830)}}
{{EmbedGHLiveSample("css-examples/flexbox/wrapping/visibility-collapse.html", '100%', 650)}}
{{EmbedGHLiveSample("css-examples/flexbox/wrapping/wrapped-visibility-collapse.html", '100%', 750)}}
{{EmbedGHLiveSample("css-examples/flexbox/order/flex-direction.html", '100%', 440)}}
{{EmbedGHLiveSample("css-examples/flexbox/order/order.html", '100%', 500)}}
{{EmbedGHLiveSample("css-examples/flexbox/order/negative-order.html", '100%', 520)}}
{{EmbedGHLiveSample("css-examples/flexbox/order/usecase-order.html", '100%', 730)}}
{{EmbedGHLiveSample("css-examples/flexbox/relationship/writing-modes.html", '100%', 360)}}
{{EmbedGHLiveSample("css-examples/flexbox/relationship/floats.html", '100%', 430)}}
{{EmbedGHLiveSample("css-examples/flexbox/relationship/flex-layout.html", '100%', 750)}}
{{EmbedGHLiveSample("css-examples/flexbox/relationship/grid-layout.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/flexbox/relationship/display-contents.html", '100%', 650)}}
{{EmbedGHLiveSample("css-examples/flexbox/use-cases/navigation.html", '100%', 550)}}
{{EmbedGHLiveSample("css-examples/flexbox/use-cases/navigation-flex.html", '100%', 550)}}
{{EmbedGHLiveSample("css-examples/flexbox/use-cases/split-navigation.html", '100%', 550)}}
{{EmbedGHLiveSample("css-examples/flexbox/use-cases/center.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/flexbox/use-cases/cards.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/flexbox/use-cases/media.html", '100%', 600)}}
{{EmbedGHLiveSample("css-examples/flexbox/use-cases/media-flipped.html", '100%', 650)}}
{{EmbedGHLiveSample("css-examples/flexbox/use-cases/label-input-button.html", '100%', 550)}}
{{EmbedGHLiveSample("css-examples/flow/block-inline/normal-flow.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/flow/block-inline/normal-flow-margin-zero.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/flow/block-inline/normal-flow-width.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/flow/block-inline/normal-flow-collapsing.html", '100%', 500)}}
{{EmbedGHLiveSample("css-examples/flow/block-inline/inline.html", '100%', 500)}}
{{EmbedGHLiveSample("css-examples/flow/block-inline/line-box.html", '100%', 500)}}
{{EmbedGHLiveSample("css-examples/flow/block-inline/flex.html", '100%', 500)}}
{{EmbedGHLiveSample("css-examples/flow/block-inline/change-formatting.html", '100%', 500)}}
```

### Motivation

Motivation is described in the project issue.

### Related issues and pull requests

- [x] https://github.com/mdn/mdn/issues/597

### CODE REVIEW

File: files/en-us/web/css/css_flexible_box_layout/typical_use_cases_of_flexbox/index.md [hunk 1]
```diff
@@ -34,23 +69,123 @@ If you wanted to respect the size property of your navigation items but have the
...
+}
+```
+
+{{EmbedLiveSample("split-navigation")}}
 
 ## Center item
 
 A long-standing joke among developers is that the hardest problem in web design is vertical centering. Vertically centering content is very straightforward with flexbox alignment properties, as the following live example shows.
 
 You can play with the alignment, aligning the item to the start with `start` or end with `end`.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/css_flexible_box_layout/typical_use_cases_of_flexbox/index.md:162]: Doesn't sound like enough of an instruction to me. Maybe something like

> Click "Play" and try playing with the alignment, for example aligning the item to the start with `start` or end with `end`

I also don't think it hurts to remind people about the "Play" button once in a while, as we aim to normalize this behavior.

File: files/en-us/web/css/css_flexible_box_layout/typical_use_cases_of_flexbox/index.md [hunk 2]
```diff
@@ -116,15 +332,90 @@ To switch the display of the media object and have the image on the right and co
...
+    <label for="text">Label</label>
+    <input id="text" type="text" />
+    <input type="submit" value="Send" />
+  </div>
+</form>
+```
+
+```css live-sample___label-input-button
+body {
+  font: 1.2em sans-serif;
```
Comment (chrisdavidmills) [inline files/en-us/web/css/css_flexible_box_layout/typical_use_cases_of_flexbox/index.md:391]: I'm not sure how much effort you are willing to spend on this, but I wonder if this example could be improved a bit. The label and submit have noticeably different font sizes, and the vertical alignment of the label is a bit off. I think you could fix the first issue without adding too much CSS, but I'm not sure about the second one.

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@bsmth a few comments for you to look at here, but nothing major. Nice work!

File: files/en-us/web/css/css_flexible_box_layout/index.md [hunk 3]
```diff
@@ -11,9 +11,36 @@ The **CSS flexible box layout** module defines a CSS box model optimized for use
...
-
-{{EmbedGHLiveSample("css-examples/flexbox/basics/simple-example.html", '100%', 600)}}
+In the following example, a container has been set to `display: flex`, which means that the three child items become flex items. The value of `justify-content` has been set to `space-between` in order to space the items out evenly on the main axis. An equal amount of space is placed between each item with the left and right items being flush with the edges of the flex container. You can also see that the items are stretching on the cross axis, due to the default value of `align-items` being `stretch`. The items stretch to the height of the flex container, making them each appear as tall as the tallest item.
+
+```html live-sample___simple-example
+<div class="box">
+  <div>One</div>
+  <div>Two</div>
+  <div>Three</div>
```
[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/web/css/css_flexible_box_layout/index.md:20]: Oh, I understand why it's there now. Makes more sense with the prose when replaced. Incoming in 61e17a9ae5

File: files/en-us/web/css/css_flexible_box_layout/typical_use_cases_of_flexbox/index.md (hunk 2 above)
Comment (bsmth) [reply files/en-us/web/css/css_flexible_box_layout/typical_use_cases_of_flexbox/index.md:391]: Agree, should be fixed in 59037c6d0d

File: files/en-us/web/css/css_flexible_box_layout/typical_use_cases_of_flexbox/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/web/css/css_flexible_box_layout/typical_use_cases_of_flexbox/index.md]: Good, added in 59037c6d0d0bb99b8d732f07ebc0bd676b6eb340

**Comment by bsmth** [comment]
Thanks for the comments, should be ready for another look

File: files/en-us/web/css/css_flexible_box_layout/typical_use_cases_of_flexbox/index.md (hunk 1 above)
Comment (chrisdavidmills) [reply files/en-us/web/css/css_flexible_box_layout/typical_use_cases_of_flexbox/index.md]: perfect

File: files/en-us/web/css/css_flexible_box_layout/typical_use_cases_of_flexbox/index.md (hunk 2 above)
Comment (chrisdavidmills) [reply files/en-us/web/css/css_flexible_box_layout/typical_use_cases_of_flexbox/index.md]: much better!

**Review by chrisdavidmills — APPROVED** [review]
LGTM!

**Comment by bsmth** [comment]
Super, thank you!

**Pull request merged**